sound/soc/sof/topology.c

```diff
@@ -166,6 +176,11 @@ static int sof_keyword_dapm_event(struct snd_soc_dapm_widget *w,
 				swidget->widget->name);
 		break;
 	case SND_SOC_DAPM_POST_PMD:
+		if (spcm->stream[stream].suspend_ignored) {
+			dev_dbg(sdev->dev, "No suspending for KWD pipelines in S0ix\n");
```

Comment:
again make the comments symmetric
PRE_PMU: "PRE_PMU event ignored, KWD pipeline is already RUNNING"
POST_PMD:  "POST_PMD even ignored, KWD pipeline will remain RUNNING"

Reply:
OK.

Reply:
comment not addressed
PRE_PMU: "PRE_PMU event ignored, KWD pipeline is already RUNNING"
POST_PMD: "POST_PMD even ignored, KWD pipeline will remain RUNNING"